superset/viz.py

```diff
@@ -170,11 +170,21 @@ def get_df(self, query_obj=None):
                 if self.datasource.offset:
                     df[DTTM_ALIAS] += timedelta(hours=self.datasource.offset)
                 df[DTTM_ALIAS] += self.time_shift
+
+            self.df_metrics_to_num(df, query_obj.get('metrics') or [])
```

Comment:
Nit: `{}.get` already takes a default value when the key is not present:

```python
self.df_metrics_to_num(df, query_obj.get('metrics', []))
```

Reply:
This way is a bit safer if the key exists with a `None` value it will make it an empty dict. That situation shouldn't occur, but it's a tiny bit safer.

Reply:
Ah, good point. I assumed keys were never `None`.